.gitignore

```
@@ -1,3 +1,6 @@
+# macOS finder meta file
+*.DS_Store
```

```suggestion
.DS_Store
```

I believe this would be effective already